lib/Controller/PageController.php

```diff
@@ -66,6 +68,14 @@ public function index() : TemplateResponse {
 			$this->eventDispatcher->dispatchTyped(new LoadEditor());
 		}
 
+		if (\OCP\Server::get(IAppManager::class)->isEnabledForUser('files') && class_exists(LoadSidebar::class)) {
```

Comment:
This should not be needed as the files app is force enabled and cannot be disabled, though maybe we just keep it if the server ever changes that